### ISSUE TITLE
Unable to load one or more of the requested types

### ISSUE THREAD

**Issue by kiwixe** [volkanceylan/Serenity#3434]
hi, i got this error, the project is new, i have installed the update 3.6.0
![captura](https://user-images.githubusercontent.com/4155477/38320584-3aeec264-37f2-11e8-9e46-4494be772856.PNG)

**Comment by brunobola** [comment]
Have you seen https://github.com/volkanceylan/Serenity/issues/3433

**Comment by kiwixe** [comment]
yes

**Comment by brunobola** [comment]
did you tried do do a reinstall with nuget command to force it to update any missed dependency ?

**Comment by kiwixe** [comment]
I used this
Update-Package -reinstall

**Comment by kiwixe** [comment]
without sense

![captura](https://user-images.githubusercontent.com/4155477/38404173-5b94f112-3925-11e8-888d-fd07754f8659.PNG)

**Comment by brunobola** [comment]
Typically a T4 transform ans rebuilb problem... Also check tooling version.

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38404323-6a5b2012-3926-11e8-84c4-18bce11200f5.PNG)

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38404330-81ac9a7a-3926-11e8-8110-19cc4142a033.PNG)

**Comment by kiwixe** [comment]
Microsoft Visual Studio Enterprise 2017 
Versión 15.6.5
VisualStudio.15.Release/15.6.5+27428.2027
Microsoft .NET Framework
Versión 4.7.02556

Versión instalada: Enterprise

Herramientas de Visual F# 10.1 para F# 4.1   00369-90000-00000-AA454
Herramientas de Microsoft Visual F# 10.1 para F# 4.1

Administrador de paquetes NuGet   4.6.0
Administrador de paquetes NuGet de Visual Studio. Para obtener más información acerca de NuGet, visite http://docs.nuget.org/.

ASP.NET and Web Tools 2017   15.0.40314.0
ASP.NET and Web Tools 2017

ASP.NET Core Razor Language Services   1.0
Provides languages services for ASP.NET Core Razor.

ASP.NET Web Frameworks and Tools 2017   5.2.51214.0
Para obtener información adicional, visite https://www.asp.net/

Azure App Service Tools v3.0.0   15.0.40215.0
Azure App Service Tools v3.0.0

Extensión de depuración de instantáneas   1.0
Información detallada de la extensión de Visual Studio de depuración de instantáneas

Herramientas comunes de Windows Azure   1.10
Ofrece servicios comunes para su uso con los servicios móviles de Microsoft Azure y Microsoft Azure Tools.

Herramientas de C#   2.7.0-beta3-62715-05. Commit Hash: db02128e6e3c4bdfc93e6ec425ac9162b4d4fe80
Componentes de C# que se usan en el IDE. Puede que se utilice una versión distinta del compilador en función de la configuración y del tipo de proyecto.

Herramientas de entrega continua de Microsoft para Visual Studio   0.3
Configuración simplificada de la integración y la entrega continuas de compilaciones desde el IDE de Visual Studio.

Herramientas de TypeScript   15.6.20202.3
Herramientas de TypeScript para Microsoft Visual Studio

Herramientas de Visual Basic   2.7.0-beta3-62715-05. Commit Hash: db02128e6e3c4bdfc93e6ec425ac9162b4d4fe80
Componentes de Visual Basic que se usan en el IDE. Puede que se utilice una versión distinta del compilador en función de la configuración y del tipo de proyecto.

Microsoft Azure Tools   2.9
Microsoft Azure Tools para Microsoft Visual Studio 2017 - v2.9.51212.2

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual Studio Tools for Containers   1.1
Develop, run, validate your ASP.NET Core applications in the target environment. F5 your application directly into a container with debugging, or CTRL + F5 to edit & refresh your app without having to rebuild the container.

Paquete del host de adaptador de depuración de Visual Studio Code   1.0
Capa de interoperabilidad para el hospedaje de adaptadores de depuración de Visual Studio Code en Visual Studio

Paquete Herramientas de Application Insights para Visual Studio   8.11.10212.1
Herramientas de Application Insights para Visual Studio

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

Servicio de lenguaje JavaScript   2.0
Servicio de lenguaje JavaScript

SQL Server Data Tools   15.1.61801.210
Microsoft SQL Server Data Tools

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38404364-c96e5024-3926-11e8-8708-f6555408ba13.PNG)

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38404380-e44b0ad6-3926-11e8-80b1-24aafa9130fb.PNG)

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38404388-fa075eec-3926-11e8-9ecc-f873c6618eb2.PNG)

**Comment by volkanceylan** [comment]
Try clearing your bin folder, remove any extra dependencies you might have

**Comment by kiwixe** [comment]
@Estrusco   I noticed that, but do not add anything

**Comment by brunobola** [comment]
@kiwixe if nothing works, try a blank project that copy the added fies and rebuild to see if it solves your problems.

**Comment by kiwixe** [comment]
this Project is blank is new completely
![captura](https://user-images.githubusercontent.com/4155477/38425586-8551b674-3971-11e8-849c-04e767943959.PNG)

**Comment by kiwixe** [comment]
![captura](https://user-images.githubusercontent.com/4155477/38426152-1e9bc4ea-3973-11e8-9efa-00c0bdc37c46.PNG)

**Comment by kiwixe** [comment]
all right, I just do this command
Update-Package Serenity.Web
and I get this
![captura](https://user-images.githubusercontent.com/4155477/38426502-14cfd9e6-3974-11e8-81b4-3f54c47aa6ff.PNG)

**Comment by ga5tan** [comment]
then update all the other stuff too. update requires officialy to update 2 things
plus there was problem and u needed to update tooling as well.
in 3.6 there is need to fix up some json file too.
best go into exception, look loadingexception and it will tell u which DLL u missing.
that I fixed out tooling (Cecil)

**Comment by brunobola** [comment]
@ga5tan I said this earlier: "Typically a T4 transform ans rebuilb problem... Also check tooling version."

You have to have more attention to comments.

**Comment by ga5tan** [comment]
@brunobola yeah u did, sorry about that. I was trying to explain to kiwixe in more depth...
